### DISCUSSION TITLE
v0.2.2-beta.2

### DISCUSSION THREAD

**Discussion by daffamumtaz2361** [cstayyab/WALC#174] [General]
Sorry for the very long delay, got very busy for a couple of months. But to worry not! To compensate, we got **features**

# What's New

- Official support for WhatsApp multi device 🥳🎉
I finally got time to update `whatsapp-web-electron.js` (the library we depend on to integrate with WhatsApp) to support multi device
- App Lock 🔒
Secure your chat from prying eyes using app lock! It's not the most secure thing in the world, but at least it will keep most people away. Currently open for suggestions, like how to reset the password (yes, there is currently no easy way to reset the password if you forgot it).  
Also, the "Lock on Screen Lock" feature is only tested on KDE. Please help me test it on other DE.
- Some tweaking to the settings page

<hr /><em>This discussion was created from the release <a href='https://github.com/WAClient/WALC/releases/tag/v0.2.2-beta.2'>v0.2.2-beta.2</a>.</em>

**Comment by albakhtari** [comment]
Is there a way of making the notifications in from WALC to show in my notification history (in KDE Plasma)?
Currently, it displays the notification for a few seconds then disappears.

**Reply by albakhtari** [reply]
I just got this error, without doing anything specific:

![Screenshot_20220416_200637](https://user-images.githubusercontent.com/79008923/163680343-016c254f-bfe1-4f0a-89da-9a2aee907c63.png)

**Reply by albakhtari** [reply]
> Is there a way of making the notifications in from WALC to show in my notification history (in KDE Plasma)?
Currently, it displays the notification for a few seconds then disappears.

Can you force electron to use `notify-send`?

![Screenshot_20220417_152020](https://user-images.githubusercontent.com/79008923/163710387-4523c1f6-375e-4a6f-82af-0a23e207579c.png)

![Screenshot_20220417_152114](https://user-images.githubusercontent.com/79008923/163710413-0faa1e72-22b1-43e7-9bfb-f2f67fd3d7d0.png)

I have no experience in electron, although I'm quite good at JS, but if there is a way to get electron to use `notify-send` then I'd like to contribute to the project and do it myself.

**Reply by daffamumtaz2361** [reply]
> I just got this error, without doing anything specific:
> 
> ![Screenshot_20220416_200637](https://user-images.githubusercontent.com/79008923/163680343-016c254f-bfe1-4f0a-89da-9a2aee907c63.png)

Probably should've added error handling there. Is it happening constantly or randomly?
